docs/src/pages/customization/typography/CustomResponsiveFontSizes.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { createMuiTheme, ThemeProvider } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+
+const theme = createMuiTheme();
+
+theme.typography.h3 = {
+  fontSize: '1.2rem',
+  '@media (min-width:600px)': {
+    fontSize: '1.5rem',
+  },
+  [theme.breakpoints.up('md')]: {
+    fontSize: '2rem',
+  },
+};
+
+export default function ResponsiveFontSizes() {
```

Comment:
`CustomResponsiveFontSizes`

Reply:
Should be, yes. It has little impact though

Reply:
Yeah, it's a minor detail. I have included in the next batch of small changes. I wonder if we shouldn't change the approach: either rename all the demos or test the exported name and throw.

Reply:
cc @mbrookes

Reply:
It tends to think that giving a nice name to the demo doesn't matter and that we can normalize it.